### DISCUSSION TITLE
Notify - adjusting position

### DISCUSSION THREAD

**Discussion by JozefPlata** [quasarframework/quasar#12284] [General - Components / Directives / etc] [answered]
Hi,

So how can I adjust the position of the Notify? Because it always fits to window size, however I'd like to be able to have some padding from the top... Is it possible?

**Comment by pikil** [comment, marked as answer]
Hi @JozefPlata,
You can try to apply css rules to the element `.q-notification`
![Screen Shot 2022-02-01 at 2 59 51 PM](https://user-images.githubusercontent.com/26779040/151902525-7d897aa1-2ccd-435a-b830-6e51b295dcfb.png)

**Comment by MilosPaunovic** [comment]
Or you could use `classes` property when you're passing config to your Nofify.

**Reply by pikil** [reply]
Ah, true as well :)

**Comment by JozefPlata** [comment]
Huh, I didn't think about it this way. It was quite easy. So I only wanted the notifications that appear from the top to have additional padding. So for that I did this:

```
.q-notifications__list--top {
    padding-top: 50px;
}
```

Thank!
